src/kvirc/ui/KviInput.cpp

```diff
@@ -87,6 +87,8 @@ KviInput::KviInput(KviWindow * pPar, KviUserListView * pView)
 	m_pHideToolsButton->setIconSize(QSize(22, 22));
 	m_pHideToolsButton->setFixedWidth(16);
 
+	m_pHideToolsButton->setToolTip("Click here to show/hide various input settings");
```

Comment:
Did you miss the `__tr2qs`? The message needs to be translateable

Reply:
Updated it.